typescript/sdk/src/deploy/proxy.ts

```diff
@@ -67,3 +72,36 @@ export async function isProxy(
   const admin = await proxyAdmin(provider, proxy);
   return !eqAddress(admin, ethers.constants.AddressZero);
 }
+
+export function proxyAdminOwnershipUpdateTxs(
+  actualConfig: Readonly<{ proxyAdmin?: DeployedOwnableConfig }>,
+  expectedConfig: Readonly<{ proxyAdmin?: DeployedOwnableConfig }>,
+  chainId: number,
+): AnnotatedEV5Transaction[] {
+  const transactions: AnnotatedEV5Transaction[] = [];
+
+  // Return early because old config files did not have the
+  // proxyAdmin property
+  if (!expectedConfig.proxyAdmin?.address) {
+    return transactions;
+  }
+
+  const actualProxyAdmin = actualConfig.proxyAdmin!;
+  assert(
+    eqAddress(actualProxyAdmin.address!, expectedConfig.proxyAdmin.address),
```

Comment:
This prevents changing of an existing proxyAdmin to another proxyAdmin. i know this is functionally different than what is used here (change proxyadmin.owner to another owner). which are we expecting? cc @paulbalaji

Reply:
I feel like both could be useful - at the top level we detect if the proxyAdmin has changed. If it's the same address but we have a different config, then we hit this function

Reply:
I agree happy to make this change!